yarn.lock

```diff
@@ -1798,6 +1798,15 @@
     opentracing "^0.14.3"
     promise-polyfill "^8.1.3"
 
+"@elastic/apm-rum-core@^5.18.0":
```

Comment:
I would consolidate this entry with the one above as the two semver's overlap

Reply:
I pushed https://github.com/elastic/kibana/pull/162722/commits/4ff1763ff93fe9655320844419d722f12470f5c1